src/qilib/utils/storage/mongo.py

```diff
@@ -71,12 +73,17 @@ def __init__(self, name: str, host: str = 'localhost', port: int = 27017, databa
             host: MongoDB host
             port: MongoDB port
             database: The database to use, if empty the name of the storage is used
+            connection_timeout: How long to try to connect to database before raising an error
```

Comment:
how about also mentioning the time unit.

Reply:
✔️